packages/@aws-cdk-testing/framework-integ/test/custom-resources/test/aws-custom-resource/integ.aws-custom-resource-athena.ts

```diff
@@ -134,9 +134,20 @@ const notebook = new AwsCustomResource(stack, 'AthenaNotebook', {
 });
 notebook.node.addDependency(workgroup);
 
-new IntegTest(app, 'CustomResourceAthena', {
+const integTest = new IntegTest(app, 'CustomResourceAthena', {
```

Comment:
Can we also add an assertion for the `AwsCustomResource` that is making a `createNotebook` call?

Reply:
Yes, will add assertion for the `notebook` as well.